### PR TITLE
My first pull request

### PR DESCRIPTION
**Pull request by Balikuddembe**
In this milestone I Implemented the following features to this project :-
Setting up Linter for HTML and CSS, I added A .gitignore file, Add A .html file includes a header tag h1 with the text "Hello Microverse!", Add .css file that includes one class with styles for your header , add A test.md file with the text "This file should be ignored by git" ,also the node_modules should be ignored with the git and not pushed to my feature branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
...
+    Description of the project.
+    Understanding the concept of version control and GitHub flow
+
+Built With
+    HTML
+    CSS
+
+    GitHub: @github/Balikuddembe
+    Twitter: @EmmanuelBaliku1
+
```
Comment (awais-amjed) [inline README.md]: Kindly use the Readme file from the [Microverse Readme Tempelate](https://github.com/microverseinc/readme-template) as described in the [Project Requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#requirements) and customize it according to your project. 🤗

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @Balikuddembe , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🎈🧨🥳

## Highlights

- [x] No Linter Issues ✔️
- [x] Correct GitHub Workflow ✔️ 
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️

- [ ] Please add a description to the Pull Request as described [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects). Shown in the image is the area where the description goes
image
- [ ] Please use a meaningful name for the title of PR that describes what the PR is related to - as described [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects) 🤗

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Kindly remember to make your commit messages meaningful as described in this [article](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8) 😊

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+Contributions to this repo are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
```
Comment (Abenezer-Tilahun) [inline README.md:33]: - [ optional ] Please try to  Update the README file , it should be descriptive and  all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a your repo.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @Balikuddembe  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Good README

almost

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] Pull Request should have a descriptive title telling you what changes are inside and should have a summary describing introduced changes in more detail Just like this
(In this milestone I Implemented the f/f :- 
  Setup linter for HTML and CSS, Add a .gitignore file, Add a .html file includes a header with the text "Hello Microverse!", Add  .css file that includes one class with styles for your header , add A test.md file with the text "This file should be ignored by git".)
 ) . Still your pull request is empty.
review

- [ ] Your Commit messages should be meaningful.(Please try to use meaningful commit messages in your future project at moment you are not required to update.)


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Balikuddembe 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**